themes/janeswalk/js/components/pages/Dashboard/DashboardTemplate.jsx

```diff
@@ -0,0 +1,18 @@
+import DashboardHeader from './DashboardHeader.jsx';
+import DashboardStore from './DashboardStore';
+import DashboardMenu from './DashboardMenu.jsx';
+
+const DashboardTemplate = ( props ) => {
+  debugger;
+  return (
+    <section>
+      <DashboardHeader {...DashboardStore.getCityData()} {...DashboardStore.getLatestPost()}/>
```

Comment:
Pass in as properties, rather than loading directly from the store, if it makes sense.